src/unit.rs

```diff
@@ -3808,6 +3812,7 @@ mod tests {
         where Endian: Endianity
     {
         let value = entry.attr_value(constants::DW_AT_name)
+            .expect("Should have parsed the name attribute")
```

Comment:
I'm slightly inclined to use `.ok().expect()` here, given that they are both considered fatal errors (and pretty similar ones too). As mentioned in the PR comment though, I think the right thing to do here is return `Result` to the caller.